themes/containerd/layouts/index.html

```diff
@@ -20,7 +20,7 @@
       <br />
 
       <div class="buttons is-centered">
-        <a class="button is-medium is-radiusless is-borderless has-text-weight-bold" href="/docs/getting-started">
+        <a class="button is-medium is-radiusless is-borderless has-text-weight-bold" href="/docs/v1.7.x/docs/getting-started/">
```

Comment:
You might want to use `/docs/latest` and define a redirect rule that refers back to the latest version.